examples/earthquake_dashboard/app.py

```diff
@@ -45,8 +44,8 @@ def get_df() -> pd.DataFrame:
 
 walker = GraphicWalker(
     df,
-    kernel_computation=True,
-    theme_key="g2",
+    server_computation=True,
```

Comment:
Does this work? I changed to `kernel_computation`. I can't see you reverting that on the `GraphicWalker`?

Reply:
The `main` branch does not work. I will make fix

![image](https://github.com/user-attachments/assets/8e3182bb-9c1d-4107-8350-a9ae217b8e8f)